composer.json

```
@@ -79,50 +79,50 @@
     "scheb/2fa-google-authenticator": "^6.0",
     "spatie/image-optimizer": "^1.6.2",
     "symfony-cmf/routing-bundle": "^3.0",
-    "symfony/cache": "^6.2",
-    "symfony/config": "^6.2",
-    "symfony/console": "^6.2",
+    "symfony/cache": "^6.4",
+    "symfony/config": "^6.4",
+    "symfony/console": "^6.4",
     "symfony/contracts": "^3.2",
-    "symfony/debug-bundle": "^6.2",
-    "symfony/dependency-injection": "^6.2",
-    "symfony/doctrine-bridge": "^6.2",
-    "symfony/doctrine-messenger": "^6.2",
-    "symfony/dom-crawler": "^6.2",
-    "symfony/error-handler": "^6.2",
-    "symfony/event-dispatcher": "^6.2",
-    "symfony/expression-language": "^6.2",
-    "symfony/filesystem": "^6.2",
-    "symfony/finder": "^6.2",
-    "symfony/framework-bundle": "^6.2",
-    "symfony/html-sanitizer": "^6.2",
-    "symfony/http-foundation": "^6.2",
+    "symfony/debug-bundle": "^6.4",
+    "symfony/dependency-injection": "^6.4",
+    "symfony/doctrine-bridge": "^6.4",
+    "symfony/doctrine-messenger": "^6.4",
+    "symfony/dom-crawler": "^6.4",
+    "symfony/error-handler": "^6.4",
+    "symfony/event-dispatcher": "^6.4",
+    "symfony/expression-language": "^6.4",
+    "symfony/filesystem": "^6.4",
+    "symfony/finder": "^6.4",
+    "symfony/framework-bundle": "^6.4",
```

See https://github.com/symfony/framework-bundle/commit/56eadf09f4c3087ecc85f07f110d7f4425a7942f
```suggestion
    "symfony/framework-bundle": "^6.4.1",
```

---

@blankse thank you for the information!